src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1632,7 +1633,25 @@ function ($a) {
                         ->arrayNode('stop_worker_on_signals')
                             ->defaultValue([])
                             ->info('A list of signals that should stop the worker; defaults to SIGTERM and SIGINT.')
-                            ->integerPrototype()->end()
+                            ->beforeNormalization()
+                                ->always()
+                                ->then(function ($v) {
+                                    return array_map(static function ($element): int|string {
```

Comment:
I would remove this `int|string` return type as this callback will run _before_ the component validates the type of value (as it validates the normalized value). We don't want a TypeError here in case the config file has an invalid value.